var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -47,14 +47,13 @@ def headers(self):
         headers.directories = [self.prefix.include.freetype2]
         return headers
 
-    def configure_args(self):
-        args = [
-            "--with-brotli=no",
-            "--with-bzip2=yes",
-            "--with-harfbuzz=no",
-            "--with-png=yes",
-            "--with-zlib=no",
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [
+            self.define("FT_DISABLE_ZLIB", True),
+            self.define("FT_DISABLE_BROTLI", True),
+            self.define("FT_DISABLE_HARFBUZZ", True),
+            self.define("FT_DISABLE_PNG", True),
```

Comment:
Why do we disable PNG? We used to enable it and still have a `libpng` dependency.

Reply:
That was me reading the wrong line. Good catch.